flake.nix

```diff
@@ -0,0 +1,199 @@
+{
+  description = "Decrypt and encrypt agenix secrets inside Emacs";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/release-22.11";
+
+    bash-strict-mode = {
+      url = "github:sellout/bash-strict-mode";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+
+    flake-utils.url = "github:numtide/flake-utils";
+
+    homeManager = {
+      url = "github:nix-community/home-manager/release-22.11";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = inputs:
+    {
+      overlays = {
+        default = final: prev: {
+          emacsPackagesFor = emacs:
+            (prev.emacsPackagesFor emacs).overrideScope'
+            (inputs.self.overlays.emacs final prev);
+        };
+
+        emacs = final: prev: efinal: eprev: {
+          agenix = inputs.self.packages.${final.system}.agenix-el;
+        };
+      };
+
+      homeConfigurations.example = inputs.homeManager.lib.homeManagerConfiguration {
+        pkgs = import inputs.nixpkgs {
+          system = inputs.flake-utils.lib.system.aarch64-darwin;
+          ## Users will refer to `inputs.agenix-el.overlays.default` instead.
+          overlays = [inputs.self.overlays.default];
+        };
+
+        modules = [
+          ./nix/home-manager-example.nix
+          {
+            # These attributes are simply required by home-manager.
+            home = {
+              homeDirectory = /tmp/agenix-el-example;
+              stateVersion = "22.11";
+              username = "agenix-el-example-user";
+            };
+          }
+        ];
+      };
+    }
+    // inputs.flake-utils.lib.eachDefaultSystem (system: let
```

Comment:
Not a huge fan on `flake-utils`, but don't have time to move it to `flake.parts` now so that's fine

Reply:
Oh, I hadn’t heard of flake.parts … I’ll have to look into that. Thanks.